src/main/java/net/jsign/PESigner.java

```diff
@@ -163,17 +176,56 @@ public void sign(PEFile file) throws Exception {
         // pad the file on a 8 byte boundary
         // todo only if there was no previous certificate table
         file.pad(8);
-        
+
         // compute the signature
         CertificateTableEntry entry = createCertificateTableEntry(file);
-        
+
         file.writeDataDirectory(DataDirectoryType.CERTIFICATE_TABLE, entry.toBytes());
         file.close();
     }
 
+    public void sign(String file) throws SignerException {

```

Comment:
This is just a shortcut for new File(String) but makes the API more unclear as PESigner.sign(String) does not tell the user that the String should be a file name. Better only have the strongly typed PESigner.sign(File).